crates/ruff_dev/src/generate_rules_table.rs

```diff
@@ -26,6 +27,9 @@ fn generate_table(table_out: &mut String, rules: impl IntoIterator<Item = Rule>,
     table_out.push('\n');
     for rule in rules {
         let status_token = match rule.group() {
+            RuleGroup::Removed => {
+                format!("<span title='Rule has been removed'>{REMOVED_SYMBOL}</span>")
+            }
```

Comment:
My vote would be to remove rules from this table entirely. What's the motivation for keeping them around?

Reply:
It seems fair to me to list the rules here when we document the removal on the details page. I do see this as obsolete if we have a versioned documentation. 

You may want to consider listing these rules *last* and designing them visually less prominent (e.g., graying them out).

Reply:
I wanted to gray them out but it actually seems like a huge pain to do it inside the markdown table. I think this will become obsolete when we do more work on the documentation but I do not think now is the time to try to build that out.